payment_api/services/payment_methods/services.go

```diff
@@ -0,0 +1,191 @@
+package pm_service // payment methods service
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+	"os"
+
+	paymentsdb "github.com/jpgsaraceni/mvp-backend/payment_api/database"
+	"github.com/jpgsaraceni/mvp-backend/payment_api/models"
+
+	"github.com/joho/godotenv"
+	"strconv"
+)
+
+func CreatePaymentMethod(paymentMethod models.PaymentMethod) (int64, error) {
+	err := godotenv.Load(".env")
+
+	if err != nil {
+		log.Fatal("Error loadinf .env file")
+	}
+
+	if paymentMethod.Name == "" {
+		return 0, &models.RequestError{Code: 422, Message: "Name is required"}
+	}
+
+	db := paymentsdb.CreateConnection()
+	defer db.Close()
+
+	sqlStatement := fmt.Sprintf(`
+		INSERT INTO %v.payment_methods (name)
+		VALUES
+			($1)
+		RETURNING id
+	`, os.Getenv("DB_SCHEMA"))
+
+	var id int64
```

Comment:
int64 parece estar em desuso, é mais int mesmo

```suggestion
	var id int
```